### PR TITLE
fix docker-compose.yml link

### PR DESCRIPTION
**Pull request by loafyloaf**

### CODE REVIEW

File: docs/deploy-with-docker.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ $ sudo docker rm gitlab
 
 ## Multiple Container deployment via Docker Compose
 
-It's easy to connect gitlab to external PostgreSQL and Redis services by making a few tweaks to the environment variables passed to the gitlab container.  In the root of this repo is a [docker-compose.yml](docker-compose.yml) file which will demonstrate a multiple container deployment that you can run locally or against a remote docker server.
+It's easy to connect gitlab to external PostgreSQL and Redis services by making a few tweaks to the environment variables passed to the gitlab container.  In the root of this repo is a [docker-compose.yml](../../docker-compose.yml) file which will demonstrate a multiple container deployment that you can run locally or against a remote docker server.
```
Comment (Tomcli) [inline docs/deploy-with-docker.md]: I think the correct link should be `../docker-compose.yml`

Comment (loafyloaf) [inline docs/deploy-with-docker.md]: yes, of course.  new patch inbound

**Comment by animeshsingh** [comment]
@loafyloaf so we are just getting the 2nd PR in - i believe we decided we dont need the first one? or do we need pieces from first one?
image

**Comment by loafyloaf** [comment]
I've closed #16

**Pull request merged**